Cargo.toml

```diff
@@ -36,6 +36,7 @@ lru = "0.12.0"
 paste = "1.0.2"
 palette = { version = "0.7.6", optional = true }
 serde = { version = "1", optional = true, features = ["derive"] }
+smallvec = "2.0.0-alpha.6"
```

Comment:
Can you please comment on why you need the alpha release version of smallvec?

Reply:
Yes lol. After about 5 hours of pain debugging lifetime errors that didn't make any sense, I discovered this: https://github.com/servo/rust-smallvec/issues/146. I'll be honest I still have pending tabs open to try and understand *why* the errors occurred, but right now the gist is that nothing compiles with the 1.0 release of smallvec.